cli/mobycli/lp_unix.go

```diff
@@ -0,0 +1,28 @@
+// +build aix darwin dragonfly freebsd linux netbsd openbsd solaris
```

Comment:
Think you can just do a `!windows` here to have the same effect

Reply:
changed